pkg/apis/openebs.io/scp/v1alpha1/storage-volume.go

```diff
@@ -117,13 +111,13 @@ type StorageVolumeStatus struct {
 	// Phase defines the state of the volume
 	Condition []StorageVolumeCondition `json:"condition,omitempty"`
 
-	// Capacity stores total, used and available size
+	// VolumeCapacity stores total, used and available size
```

Comment:
Type can be VolumeCapacity and field name can be just Capacity, refer the pool CR.